flaml/automl.py

```diff
@@ -1975,6 +2176,33 @@ def custom_metric(
                 the automl constructor, flaml will automatically (and under the hood)
                 add it as an additional element in the metric_constraints. Essentially 'pred_time_limit'
                 specifies a constraint about the prediction latency constraint in seconds.
+            custom_hp: dict, default=None | The custom search space specified by user
+                Each key is the estimator name, each value is a dict of the custom search space for that estimator. Notice the
+                domain of the custom search space can either be a value of a sample.Domain object.
+        ```python
```

Comment:
Please try to compile the documentation website locally and verify no extra newline is required before this line.
Or, use a newline before this line and other similar cases.

Reply:
how to compile?